nipype/algorithms/compcor.py

```diff
@@ -22,6 +22,15 @@ class CompCoreOutputSpec(TraitedSpec):
 class CompCore(BaseInterface):
     '''
     Node with core CompCor computation, used in aCompCor and tCompCor
+
+    Example
+    -------
+
+    >>> ccinterface = CompCore()
```

Comment:
You need to add this on top of the file: https://github.com/nipy/nipype/blob/master/nipype/interfaces/afni/preprocess.py#L5 for the doctests to run independently of CWD